part-1-ajax/test.js

```diff
@@ -1,76 +1,67 @@
 function getCountryName(code) {
-    $.ajax({
-      //uses the three letter code to get the country's full name
-      }
-    })
-  }
+    // console.log(code);
+    console.log(code.name);
+    $("ul").append("<li>" + code.name + "</li>")
+}
 
-  function parseData(country) {
+function parseData(country) {
     //takes the country data
+    console.log(country);
+
     //extracts the name, capital, population, and neighboring countries
+    var name = country[0].name;
+    console.log('name: ', name)
```

Comment:
definitely good habit to have - console logging stuff. might what to take it out at the end or comment it out.